themes/qgis-theme/forusers_download.html

```diff
@@ -35,61 +35,38 @@ <h4>{{ _('Download for Windows') }} <i class="icon-chevron-down pull-right"></i>
-                              <li></li>
-                              <li class="download-text">sha256</li>
-                          </ul>
+                        <a class="btn btn-large" href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ release|e }}-{{ binary|e }}.msi">
+                          <div class="primary-download-button">
+                            <div class="{{ "qgis{}-download".format( "2" if version.startswith("2.") else "" ) }}"></div>
+                            <div class="download-text">{{ _('QGIS Standalone Installer Version %s')|format( version ) }}</div>
+                          </div>
                         </a>
-                        <h6>{{ _('Long term release (most stable):') }}</h6>
-                        <a href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ ltrrelease|e }}-{{ ltrbinary|e }}.msi" class="reference external">
-                          <ul class="inline download-row">
-                              <li><i class="icon-download-alt icon-large"></i></li>
-                              <li class="{{ "qgis{}-download".format( "2" if ltrversion.startswith("2.") else "" ) }}"></li>
-                              <li class="download-text">{{ _('QGIS Standalone Installer Version %s')|format( ltrversion ) }}</li>
-                          </ul>
+                        <a class="secondary-download-link" href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ ltrrelease|e }}-{{ ltrbinary|e }}.msi">
+                          {{ _('QGIS Standalone Installer Version %s')|format( ltrversion ) }} LTR
                         </a>
-                        <a href="https://qgis.org/downloads/QGIS-OSGeo4W-{{ ltrrelease|e }}-{{ ltrbinary|e }}.sha256sum" class="reference external button">
-                          <ul class="inline download-row">
-                              <li></li>
-                              <li></li>
-                              <li class="download-text">sha256</li>
-                          </ul>
-                        </a>
-			<p>{{ _('Note that the MSI installers are much bigger than the previous installers.  This is because they include significant larger packages (eg. PROJ 8).  The main reason for the switch to MSI were the size limits previously used NSIS has, which was blocking updates of dependencies.') }}</p>
+                        <p><i>{{ _('Standalone installers are recommended for new users.') }} </i></p>
+                      
+                      </div>
+                      <div class="card">
+                        <div>
+                          <a class="btn btn-large" href="https://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe">
+                            <div class="primary-download-button">
+                              <div class="{{ "qgis{}-download".format( "2" if version.startswith("2.") else "" ) }}"></div>
+                              <div class="download-text">{{ _('OSGeo4W Network Installer') }}</div>
+                            </div>
```

Comment:
```suggestion
                              <div class="{{ "qgis{}-download".format( "2" if version.startswith("2.") else "" ) }}"></div>
                              <div class="download-text">{{ _('OSGeo4W Network Installer') }}</div>
```

I'd suggest to stick to one single download button (and make the other one a link)

Reply:
Thanks for improvements, wil have a close look soon